cli/log_sink.go

```diff
@@ -0,0 +1,98 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/url"
+
+	"github.com/flynn/flynn/controller/client"
+	ct "github.com/flynn/flynn/controller/types"
+	"github.com/flynn/go-docopt"
+)
+
+func init() {
+	register("log-sink", runLogSink, `
+usage: flynn log-sink
+       flynn log-sink add syslog <url> [<prefix>]
+       flynn log-sink remove <id>
```

Comment:
Should these commands be scoped to the cluster (under the `flynn cluster` subcommand or via a mandatory flag) to allow room for per-app sinks?